cli/src/cluster_query.rs

```diff
@@ -1044,6 +1075,42 @@ pub fn process_leader_schedule(
     }))
 }
 
+pub fn process_get_recent_priority_fees(
+    rpc_client: &RpcClient,
+    config: &CliConfig,
+    accounts: &[Pubkey],
+    num_slots: Option<Slot>,
+) -> ProcessResult {
+    let fees = rpc_client.get_recent_prioritization_fees(accounts)?;
+    let mut min = u64::MAX;
+    let mut max = 0;
+    let mut total = 0;
+    let num_slots = num_slots
+        .unwrap_or(fees.len() as u64)
+        .min(fees.len() as u64)
+        .max(1) as usize;
```

Comment:
this can all be enforced with `clap`, which hands the user nice guidance instead of doing things silently behind their back

Reply:
I have to parse again in an .validator() right? kind of annoying to do, but not perf sensitive and gives better feedback. It's just for the 0 case.

Reply:
> I have to parse again in an .validator() right?

Yeah. And it is kind of annoying 😅 Honestly, if someone is foolish enough to pass `--limit-num-slots 0`, I'm okay with just returning empty/default data. But Trent may feel differently